File: exercises/beecrowd/1149/1149.go
// https://judge.beecrowd.com/pt/problems/view/1149
package main

import "fmt"

func main() {
	var a, n int
	fmt.Scan(&a)

	for {
		fmt.Scan(&n)
		if n > 0 {
			break
		}
	}

	sum := 0
	for i := 0; i < n; i++ {
		sum += a + i
	}

	fmt.Println(sum)
}

File: exercises/beecrowd/1149/1149.ts
// https://judge.beecrowd.com/pt/problems/view/1149
import * as fs from "fs";

function main(): void {
  const values = fs
    .readFileSync("/dev/stdin", "utf8")
    .trim()
    .split(/\s+/)
    .map(Number);

  const a = values[0];
  const n = values.find((v, i) => i > 0 && v > 0)!;

  let sum = 0;
  for (let i = 0; i < n; i++) {
    sum += a + i;
  }

  console.log(sum);
}

main();
